src/pydata_sphinx_theme/assets/styles/content/_toctree.scss

```diff
@@ -0,0 +1,33 @@
+/**
+ * Style the toctree component in pages (avoid modifying the navbars)
+ */
+.toctree-wrapper {
+  p.caption {
+    font-size: 1.5em;
+    margin-bottom: 0em;
+  }
+
+  & > ul {
+    padding-left: 0;
+  }
+
+  .toctree-l1,
+  .toctree-l2,
+  .toctree-l3,
+  .toctree-l4,
+  .toctree-l5,
+  .toctree-l6 {
+    list-style: none;
+    margin-bottom: 0.5em;
+
+    & > a {
+      list-style: none;
+      font-size: 1.3em;
```

Comment:
Is there some way that we could make each successive level past the top level a bit smaller? (or even just everything under the top level at the same smaller font)